packages/node/src/yargs.ts

```diff
@@ -344,6 +328,13 @@ export const yargsOptions = yargs(hideBin(process.argv))
         'File based only : local path of the merkle mountain range (.mmr) file',
       type: 'string',
     },
+    'mmr-store-type': {
+      demandOption: false,
+      describe: 'Store MMR in either a file or a postgres DB',
+      type: 'string',
+      choices: ['file', 'postgres'],
+      default: 'postgres',
```

Comment:
I think it would be worth updating the default in NodeConfig too, in case a config file is used